web/themes/contrib/civictheme/civictheme.post_update.php

```diff
@@ -619,3 +620,97 @@ function civictheme_post_update_enable_focal_point_configurations_2(): void {
   ];
   \Drupal::classResolver(CivicthemeUpdateHelper::class)->deleteConfig($old_field_configs);
 }
+
+/**
+ * Updates blocks from 'sidebar' region to 'sidebar_top_left'.
+ *
+ * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_move_blocks_to_sidebar_top_left(): void {
+  $themes = ['civictheme'];
+  $active_theme = \Drupal::theme()->getActiveTheme();
+  $active_theme_name = $active_theme->getName();
+  if ($active_theme_name !== 'civictheme') {
+    $themes[] = $active_theme_name;
+  }
+
+  foreach ($themes as $theme) {
+    $blocks = \Drupal::entityTypeManager()
+      ->getStorage('block')
+      ->loadByProperties(['theme' => $theme, 'region' => 'sidebar']);
+    if ($blocks) {
+      foreach ($blocks as $block) {
+        $block->setRegion('sidebar_top_left');
+        $block->set('status', TRUE);
+        $block->save();
+      }
+    }
+    // Check if <theme_name>_side_navigation block exists.
+    $sideNavigationBlock = \Drupal::entityTypeManager()
+      ->getStorage('block')
+      ->loadByProperties(['theme' => $theme, 'id' => $theme . '_side_navigation']);
+    if ($sideNavigationBlock) {
+      foreach ($sideNavigationBlock as $block) {
+        // Update the region to 'sidebar_top_left'.
+        $block->setRegion('sidebar_top_left');
+        $block->set('status', TRUE);
+        $block->save();
+      }
+    }
+  }
+
+  \Drupal::messenger()->addMessage('All blocks in the "sidebar" region have been moved to "sidebar_top_left".');
+}
+
+/**
+ * Enables "3 column" layout for the Page content type.
+ * 
+ * @SuppressWarnings(PHPMD.ElseExpression)
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_enable_three_column_layout(): void {
+  // Load the current display configuration for the Page content type.
+  $entity_display = \Drupal::entityTypeManager()
+    ->getStorage('entity_view_display')
+    ->load('node.civictheme_page.default');
+
+  // Check if the display is already set to use the "3 column" layout.
+  if ($entity_display) {
+    $entity_view_mode_restriction = $entity_display->getThirdPartySetting('layout_builder_restrictions', 'entity_view_mode_restriction', []);
+    if (!empty($entity_view_mode_restriction['allowed_layouts']) && !in_array('civictheme_three_columns', $entity_view_mode_restriction['allowed_layouts'])){
+      
+      $update_config = TRUE;
+      if (!empty($entity_view_mode_restriction['allowlisted_blocks'])) {
+        $update_config = FALSE;
+      }
+
+      if ($update_config){
+
+        // Update the configuration to use the "3 column" layout.
+        $entity_view_mode_restriction['allowed_layouts'] = [];
+        $entity_view_mode_restriction['allowed_layouts'][] = 'civictheme_three_columns';
```

Comment:
please add removal of `civictheme_onecol*` from the list of the allowed layouts.